Program.cs

```diff
@@ -721,103 +722,117 @@ public static void Props()
             Pause();
             Console.Clear();
         }
-
-
-        public static void attack()
+            
+        //This is the attack method. I have reworked it so that there is a global player health variable. This means the method can be used again if needed. It also takes input for the computer health now
+        //The idea is at base level it is simply attack and defence with a higher roll needed to hit if you are attacking someone defending. You can also pick up the knife or the crowbar which increase your damage
+        public static void Attack(int compHP)
         {
-            string answer;
-            int playeratt, playerHP, computeratt, computerHP, computer;
-            playeratt = 5;
-            playerHP = 25;
-            computerHP = 25;
-
+            bool done = false;
+            int i = 0;
+            string weapon;
+            string temp;
+            int playerDMG = 0;
+            int compDMG = 4;
+            int computerAction;
+            int toHit;
             Random rand = new Random();
-            computer = rand.Next(5);
-
-            switch (computer)
+            do //The loop for checking if there is a weapon in players inventory. Both weapons do the same damage so the first one found is used
             {
-                case 0:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 1:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 2:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 3:
-                    Console.WriteLine("computer attack");
-                    break;
-                case 4:
-                    Console.WriteLine("computer defense");
-                    break;
-            }
-
-
-            while ((computerHP > 0) && (playerHP > 0))
+                if (inventory[i] == "crowbar" || inventory[i] == "knife")
```

Comment:
Cool idea to be able to use the items in your inventory but it would be cool to add all of the items that are in the inventory to the list as well.

Reply:
Added fire extinguisher to weapon list. The other items wouldn't make sense as weapons.